apps/src/utils.js

```diff
@@ -110,7 +110,9 @@ exports.stripQuotes = function(inputString) {
  * Defines an inheritance relationship between parent class and this class.
  */
 Function.prototype.inherits = function (parent) {
-  this.prototype = _.create(parent.prototype, { constructor: parent });
+  this.prototype = Object.create(parent.prototype);
+  this.prototype.constructor = this;
+  this.superPrototype = parent.prototype;
```

Comment:
I wonder if we should just expose super, and let others do what they want with it (i.e. beeItemDrawer would then use BeeItemDrawer.super.prototype. I also wonder if super or superPrototype should have some underscores in it (though I'm not sure exactly what the convention is). I would guess something like this.super_ or this.__super.

I guess another option would be to not use superPrototype at all and have this.__base = DirtDrawer.prototype.